04.class/memo_app.js

```diff
@@ -97,4 +95,9 @@ export class MemosApp {
       callback(Object.values(answer.result)[0]);
     })();
   }
+
+  extractFirstLine(baseData) {
+    const lines = baseData.split("\n");
+    return lines[0];
```

Comment:
```suggestion
    return baseData.split("\n")[0];
```

としてもいいかもしれません（好みの問題）

Reply:
ご提案いただいた形に修正いたしました！